db/migrate/20190516102219_remove_classic_car_value_from_other_assets_declarations.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveClassicCarValueFromOtherAssetsDeclarations < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :other_assets_declarations, :classic_car_value, :decimal
```

Comment:
The two vehicle columns were already removed by this merged migration: https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/507/commits/90c91d1fbbd3a2de7c1bf3ca0063161e077177f5
(That might be what is causing the UAT deploy to fail)

Reply:
deleted this file